src/frontend/src/components/groups/GroupRow/GroupRow.tsx

```diff
@@ -0,0 +1,117 @@
+import { Avatar, Box, Card, CheckBox, Grid, Text } from 'grommet';
+import React, { ChangeEvent, useRef, useState } from 'react';
+import { More } from 'grommet-icons';
+import { useMediaQuery } from 'react-responsive';
+
+export interface Group {
+  id: string;
+  name: string;
+  members: {
+    id: string;
+    name: string;
+    avatar: string;
+  }[];
+}
+
+export interface GroupRowProps {
+  /**
+   * The group to display
+   */
+  group: Group;
+  /**
+   * Callback when the group is selected or deselected
+   */
+  onToogle: (selected: boolean) => void;
+  /**
+   * Is the group currently selected
+   */
+  selected: boolean;
+}
+
+export default function GroupRow(props: GroupRowProps) {
+  const { group, onToogle, selected } = props;
+  const ref = useRef<HTMLInputElement>(null);
+
+  const handleToogle = (event: ChangeEvent<HTMLInputElement>) => {
+    onToogle(event.target.checked);
+  };
+
+  // Get a number between 0 and 3 representing the size of the screen
+  // 0 = big, 1 = large, 2 = medium, 3 = small, 4 = xsmall
+  const isBigScreen = useMediaQuery({ query: '(min-width: 1824px)' });
+  const isLargeScreen = useMediaQuery({ query: '(min-width: 1224px)' });
+  const isMediumScreen = useMediaQuery({ query: '(min-width: 824px)' });
+  const isSmallScreen = useMediaQuery({ query: '(min-width: 424px)' });
+  const matchingScreenSizes = [isBigScreen, isLargeScreen, isMediumScreen, isSmallScreen, true];
+  const screenSize = matchingScreenSizes.findIndex((size) => size);
+  const numberOfDisplayedMembers = 9 - 2 * screenSize;
```

Comment:
I don't know if all of this is not overkill. First grommet contains a responsive context you can use. You should also use its provider to make you app responsive.

Reply:
Yes, maybe. I'll look at what grommet provide... If it exposes the theme-defined breakpoints it would be much better indeed !